.github/workflows/linux.yml

```diff
@@ -128,7 +128,13 @@ jobs:
     - name: Install
       run: |
         sudo apt update
+        # Remove the default Python and link python3.11 in its stead to avoid
+        # compatibility issues between the two versions.
+        sudo apt -y autoremove python3
         sudo apt install clang-14 cmake gfortran libhdf5-dev python3.11 python3.11-dev wget
+        python="$(which python3.11)"
+        sudo ln -s "$python" "${python%.11}"
+        sudo ln -s "$python" "${python%3.11}"
```

Comment:
Is this still needed with a venv?

Reply:
I'm not using a venv here, but I can try it as an alternative

Reply:
Yep, that works too. I just have to take care that Spack does not use the venv since it will try finding headers inside the venv.